tap_dynamodb/sync_strategies/full_table.py

```diff
@@ -101,3 +116,71 @@ def sync(config, state, stream):
     singer.write_version(table_name, stream_version)
 
     return rows_saved
+
+def get_expr_names(projection_element, expression_list):
+    '''
+    Prepare expression names for general expression attributes as well as for each of the nested
+    expression attributes.
+    For example:
+    projections = ["Comment", "Ticket"], expressions = ["Comment"]
+    return = "#Commnt"
+    Example of nested expression :
+    projections = ["Name[0].Comment"], expressions = ["Name.Comment"]
+    return = "#Namme[0]" and "#Commnt" in the second loop call
+    '''
+    half_length = int(len(projection_element)/2)
+    expr = "#{}".format(projection_element[:half_length+1]+projection_element[-2:]) # prepare expression attribute name
```

Comment:
@namrata270998 Please simplify the expression as in,
"Projection": "Comment", "Expression": "#Comment"

Reply:
Done